tests/acceptance/features/apiTestingApp/loginDate.feature

```diff
@@ -0,0 +1,54 @@
+@api
+Feature: get and set the last login date
+
+  Background:
+    Given these users have been created without skeleton files:
+      | username |
+      | user1    |
+
+  Scenario Outline: Get the last login date of a user
+    Given using OCS API version "<ocs-api-version>"
+    When the administrator gets the last login date for user "user1" using the testing API
+    Then the last login date for the user should be "0" days ago
+    Then the HTTP status code should be "<ocs-status-code>"
+    And the OCS status code should be "<http-status-code>"
+    Examples:
+      | ocs-api-version | ocs-status-code | http-status-code |
+      | 1               | 200             | 200              |
+      | 2               | 200             | 200              |
+
+  Scenario Outline: Set the last login date of a user
+    Given using OCS API version "<ocs-api-version>"
+    When the administrator sets the last login date for user "user1" to "7" days ago using the testing API
+    And the administrator gets the last login date for user "user1" using the testing API
+    Then the last login date for the user should be "7" days ago
```

Comment:
Since this step is dependent on step `And the administrator gets the last login date for user "user1" using the testing API` there should be mention of something like last response in the step.

for eg. `Then the last login date in the response should be "7" days ago`